package.json

```diff
@@ -30,7 +30,7 @@
     "build": "gatsby build",
     "develop": "gatsby develop",
     "dev": "yarn develop",
-    "format": "prettier-eslint --write 'src/**/*.js'",
+    "format": "prettier-eslint --write '**/*.js'",
```

Comment:
I figure it makes sense to lint config files too. I added a `.eslintignore` file as well to allow for ignoring directories that sit in the root